src/OrchardCore/OrchardCore.Recipes.Core/Services/RecipeHarvester.cs

```diff
@@ -51,7 +47,7 @@ protected async Task<IEnumerable<RecipeDescriptor>> HarvestRecipesAsync(string p
             var recipeDescriptors = new List<RecipeDescriptor>();
 
             var recipeFiles = _hostingEnvironment.ContentRootFileProvider.GetDirectoryContents(path)
-                .Where(x => !x.IsDirectory && x.Name.EndsWith(".recipe.json", StringComparison.Ordinal));
+                .Where(f => !f.IsDirectory && f.Name.EndsWith(_recipeExtension, StringComparison.OrdinalIgnoreCase));
```

Comment:
`Ordinal` is different than `OrdinalIgnoreCase`. I don't think we should change this behavior without some rationale.

Reply:
I'm not sure why the case is matter here?

Reply:
The file should be named something.recipe.json, since this is everything in Orchard assumes. So, if we change the behavior here, then there will be a disconnect between this class and the rest of the system.

Reply:
This was case sensitive, anyhow I revert this change back

Reply:
Yes, and I'm arguing that it needs to remain case-sensitive (or you need to change it in every place, but I don't see the point).

Reply:
OK, leave it like that, then.